plugin/trino-mongodb/src/test/java/io/trino/plugin/mongodb/MongoServer.java

```diff
@@ -25,7 +25,7 @@
 
     public MongoServer()
     {
-        this("4.0.0");
+        this("4.2.0");
```

Comment:
Please add a prepratory commit as "Upgrade minimum supported version to 4.2 in MongoDB". The commit should contain docs change of "Requirements".

Reply:
https://trinodb.slack.com/archives/CP1MUNEUX/p1668702038526179